examples/two-columns-layout/src/app.tsx

```diff
@@ -137,15 +144,37 @@ autocomplete({
         className="aa-PanelLayout aa-Panel--scrollable"
         onMouseLeave={onMouseLeave}
       >
+        {!hasResults && (
+          <div className="aa-NoResultsQuery">
+            Sorry, we didn't find any matches for "{state.query}"
+          </div>
+        )}
+
         <div className="aa-PanelSections">
           <div className="aa-PanelSection--left">
-            {hasResults && (
-              <div>
-                {recentSearches}
-                {querySuggestions}
-                {categories}
-                {brands}
-                {faq}
+            {hasResults ? (
+              (!state.query && recentSearches) ||
+              (state.query && (
+                <div>
+                  {recentSearches}
+                  {querySuggestions}
+                  {categories}
+                  {brands}
+                  {faq}
+                </div>
+              ))
+            ) : (
+              <div className="aa-NoResultsAdvices">
+                <div className="aa-NoResultsAdvicesTitle">
+                  Try the following:
+                </div>
+                <ul className="aa-NoResultsAdvicesList">
+                  <li>Double check your spelling</li>
+                  <li>Use fewer keywords</li>
+                  <li>
+                    Search fo an item that is less specific and refine results
```

Comment:
The "refine results" part is confusing, no?

```suggestion
                    Search fo a less specific item
```